src/js/modules/infragistics.ui.editors.js

```diff
@@ -11124,6 +11128,11 @@
 					}
 				};
 			}
+			options.onChangeMonthYear  = function () {

```

Comment:
@Lipata If we handle jQuery datepicker events internally (like `onSelect`) those should save a possible such handler passed in with `datepickerOptions` by the user so those are properly called as well